interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +126,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$edit && $GLOBALS['enable_group_therapy'] )?' disabled=true; ':'';?>
+<?php if($disabled) echo '<script>$( document ).ready(function(){
```

Comment:
Oh I see! You have conditionally modified the global setting $GLOBALS['enable_group_therapy'] above. This is confusing because global settings are not expected to be used as local variables.